app/conversations/schemas.py

```diff
@@ -0,0 +1,17 @@
+from marshmallow import fields, validate
+
+from app.common.schemas import CamelCaseSchema, ma
+from app.conversations.enums import ConversationStatus
+from app.models import Conversations
+
+
+class ConversationEditSchema(CamelCaseSchema, ma.SQLAlchemySchema):
+    class Meta:
+        model = Conversations
+        load_instance = True
+
+    conversation_id = ma.auto_field("conversation_uuid")
+    receiver_name = ma.auto_field()
```

Comment:
direct map from the DB field, we could map all the fields using `ma.SQLAlchemyAutoSchema`